flow360/component/flow360_params/flow360_output.py

```diff
@@ -0,0 +1,323 @@
+    nu_hat: Optional[bool] = pd.Field(alias="nuHat")
+    primitive_vars: Optional[bool] = pd.Field(alias="primitiveVars")
+    q_criterion: Optional[bool] = pd.Field(alias="qcriterion")
+    residual_navier_stokes: Optional[bool] = pd.Field(alias="residualNavierStokes")
+    residual_transition: Optional[bool] = pd.Field(alias="residualTransition")
+    residual_turbulence: Optional[bool] = pd.Field(alias="residualTurbulence")
+    s: Optional[bool] = pd.Field()
+    solution_navier_stokes: Optional[bool] = pd.Field(alias="solutionNavierStokes")
+    solution_turbulence: Optional[bool] = pd.Field(alias="solutionTurbulence")
+    solution_transition: Optional[bool] = pd.Field(alias="solutionTransition")
+    T: Optional[bool] = pd.Field(alias="T")
+    vorticity: Optional[bool] = pd.Field()
+    wall_distance: Optional[bool] = pd.Field(alias="wallDistance")
+    low_numerical_dissipation_sensor: Optional[bool] = pd.Field(
+        alias="lowNumericalDissipationSensor"
+    )
+    residual_heat_solver: Optional[bool] = pd.Field(alias="residualHeatSolver")
+
+
+OutputFormat = Union[Literal["paraview"], Literal["tecplot"], Literal["both"]]
+
+
+class Surface(Flow360BaseModel):
+    """:class:`Surface` class"""
+
+    output_fields: Optional[List[str]] = pd.Field(alias="outputFields")
+
+
+class _GenericSurfaceWrapper(Flow360BaseModel):
+    """:class:`_GenericSurfaceWrapper` class"""
+
+    v: Surface
+
+
+class Surfaces(Flow360SortableBaseModel):
+    """:class:`Surfaces` class"""
+
+    @classmethod
+    def get_subtypes(cls) -> list:
+        return [_GenericSurfaceWrapper.__fields__["v"].type_]
+
+    # pylint: disable=no-self-argument
+    @pd.root_validator(pre=True)
+    def validate_surface(cls, values):
+        """
+        root validator
+        """
+        return _self_named_property_validator(
+            values, _GenericSurfaceWrapper, msg="is not any of supported surface types."
+        )
+
+
+class SurfaceOutput(OutputBase):
+    """:class:`SurfaceOutput` class"""
+
+    output_format: Optional[OutputFormat] = pd.Field(alias="outputFormat")
+    animation_frequency: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(
+        alias="animationFrequency"
+    )
+    animation_frequency_offset: Optional[int] = pd.Field(alias="animationFrequencyOffset")
+    animation_frequency_time_average: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(
+        alias="animationFrequencyTimeAverage"
+    )
+    animation_frequency_time_average_offset: Optional[int] = pd.Field(
+        alias="animationFrequencyTimeAverageOffset"
+    )
+    compute_time_averages: Optional[bool] = pd.Field(alias="computeTimeAverages")
+    velocity_relative: Optional[bool] = pd.Field(alias="VelocityRelative")
+    write_single_file: Optional[bool] = pd.Field(alias="writeSingleFile")
+    primitive_vars: Optional[bool] = pd.Field(alias="primitiveVars")
+    start_average_integration_step: Optional[bool] = pd.Field(alias="startAverageIntegrationStep")
```

Comment:
move all bool options to separate class that will be only used in updater but not be part of allowed configuration

Reply:
Fixed in 05ca5e6